github-webhooks-to-s3/README.md

```diff
@@ -0,0 +1,23 @@
+<!-- START -->
```

Comment:
Might also be valuable to write a short description of what this is and an example of how to use it